payjoin-cli/src/app/v2.rs

```diff
@@ -389,3 +425,28 @@ impl ReceiveStore {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[tokio::test]
+    async fn test_fetch_ohttp_keys() {
+        let relay_port = find_free_port();
+        let relay_url = Url::parse(&format!("http://0.0.0.0:{}", relay_port)).unwrap();
+        let pj_endpoint = Url::parse("https://payjo.in:443").unwrap();
+        tokio::select! {
```

Comment:
I saw this `tokio::select!` few times already and always get confused about it. can please explain whats the motivation of using it in this kind of scenarios?

Reply:
Select is a way to spin up call concurrent async methods without starting new tasks. It's less boilerplate when multiple async calls, like multiple servers, need to run in e.g. tests. It also cancels all of the functions as soon as one of them returns.

https://tokio.rs/tokio/tutorial/select